R/force_sorted.R

```diff
@@ -0,0 +1,71 @@
+#' @title Forcing a numeric vector into a monotonously increasing sequence.
+#'
+#' @description
+#' This function performs interpolation on the non-increasing parts of an
+#' input vector to ensure its values are monotonously increasing.
+#' If the values are non-increasing at the end of the vector, these values
+#' will be monotonously increasing by the value defined with parameter `by `
```

Comment:
maybe better here *... these values will be replaced by a sequence of numeric values, starting from the last increasing value in the input vector, and increasing by a very small value, which can be defined with parameter `by`. *